resources/views/declarations/index.blade.php

```diff
@@ -66,16 +63,28 @@
 						<td>{{ $declaration->date->format('Y-m-d') }}</td>
 						<td>
 							@unless ($declaration->filename === null || $declaration->filename == '')
-								<a href="{{ asset('uploads/declarations/' . $member->id . '/' . $declaration->filename) }}" target="_blank">{{$declaration->filename}}</a>
+								<a href="{{ url('/declarations', [$declaration->id, 'file']) }}" target="_blank">{{$declaration->original_filename}}</a>
 							@else
 								-
 							@endunless
 						</td>
 						<td>{{ $declaration->description }}</td>
-						<td>{{ formatPrice($declaration->amount) }}</td>
+						<td>{!! formatPrice($declaration->amount) !!}</td>
```

Comment:
Kan deze niet ook vervangen worden door `@money()`?

Reply:
Yes I find/replaced them, but I must have missed this one